xAPI.md

```diff
@@ -2475,9 +2468,15 @@ identifiers other than Activity id.
 
 * Metadata MAY be provided with an identifier.
 * If metadata is provided, both name and description SHOULD be included.
-* IRIs SHOULD be defined within a domain controlled by the [Metadata Provider](#def-metadata-provider) creating the IRI.
+* IRIs SHOULD* be defined within a domain controlled by the [Metadata Provider](#def-metadata-provider) creating the IRI or granting permission
```

Comment:
I've been thinking through this section, and really it shouldn't talk about the domain here, but about the IRI itself -- that's what ownership or permission is needed for. "Metadata Providers defining new IRIs SHOULD* only use IRIs they control or have permission from the controller to use." Both 1) not all IRIs are domain-based, and 2) domain ownership is just a part of determining IRI ownership.